.github/workflows/docker_publish.yaml

```diff
@@ -6,14 +6,8 @@ on:
       - main
     paths-ignore:
       - website/**
-    tags:
-      - 'v*'
+  workflow_call:
   workflow_dispatch:
-    inputs:
-      ignore-push:
-        description: 'Ignore push to dockerhub. If not set the image will be pushed with the sha of the commit as tag'
-        required: false
-        type: boolean
```

Comment:
Removed the ability of ignoring pushes. We never use it, it was mainly for testing this workflow during development